cmd/nocc-daemon/main.go

```diff
@@ -69,6 +69,8 @@ func main() {
 		"", "NOCC_LOG_FILENAME")
 	logVerbosity := common.CmdEnvInt("Logger verbosity level for INFO (-1 off, default 0, max 2).\nErrors are logged always.", 0,
 		"", "NOCC_LOG_VERBOSITY")
+	connectTimeout := common.CmdEnvInt("Timeout, in milliseconds, to connect to servers at start up (default 3000).\nIf any server is unreachable, all invocations pointing to that server fall back to local compilation.", 3000,
+		"", "NOCC_CONNECT_TIMEOUT")
```

Comment:
BTW — maybe, just to remote this option? It was added before tmpfs, because we thought, that 3 seconds weren't enough to connect — but actually, it was for Mkdir. Now it's solved, maybe this option is not needed at all?